### DISCUSSION TITLE
avoid showing shipping process

### DISCUSSION THREAD

**Discussion by micromotores** [PHPMailer/PHPMailer#2357] [General]
I want to avoid showing the process of sending the attached server I am going to put the image of what I do not want to appear while it performs the sending process
![image](https://user-images.githubusercontent.com/79323549/121708719-782fb200-ca9d-11eb-89e1-6902068e9424.png)
![image](https://user-images.githubusercontent.com/79323549/121708810-8aa9eb80-ca9d-11eb-9b92-ee10ef9b9df9.png)

**Comment by micromotores** [comment]
ya logre quitarlo gracias :)

**Comment by Synchro** [comment]
$mail->SMTPDebug = 0;
